spec/swagger_helper.rb

```diff
@@ -126,6 +141,46 @@
               qris_rating: { type: :string, example: '1' }
             }
           },
+          case_cycle: {
+            type: :object,
+            properties: {
+              case_number: {
+                anyOf: [
+                  { type: :string, example: '123-45' },
+                  { type: :null }
+                ]
+              },
+              copay_cents: { type: :integer, minimum: 0 },
+              copay_currency: { type: :string, example: 'USD' },
+              copay_frequency: {
+                anyOf: [
+                  { '$ref': '#/components/schemas/copay_frequency' },
```

Comment:
```suggestion
                  { '$ref': '#/components/schemas/copay_frequencies' },
```

Reply:
👍  I need to change the name of in the schema, too.